resources/assets/components/PagingButtons/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+class PagingButtons extends React.Component {
+
+  render() {
+    const prev = this.props.prev;
+    const next = this.props.next;
+    const prevCopy = this.props.prevCopy;
+    const nextCopy = this.props.nextCopy;
+
+    return (
+        <div className="container__block">
+			<a href={prev}>{prevCopy}</a>
+			<div className="next-page">
```

Comment:
The best I could figure out was how it is now, which only shows next or previous if there is a next or previous. Disabling the link would probably be easiest, I was trying to figure out a better way to hide but this is what I came up with